kolibri/plugins/coach/assets/src/views/common/QuizStatusModal.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+
+  <KModal
+    :title="modalHeader"
+    :submitText="coreString('continueAction')"
+    :cancelText="coreString('cancelAction')"
+    @cancel="$emit('cancel')"
+    @submit="$emit('submit')"
+  >
+    <div>{{ modalDetail }}</div>
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+  export default {
+    name: 'QuizStatusModal',
+    mixins: [commonCoreStrings],
+    props: {
+      modalHeader: {
+        type: String,
+        required: false,
+        default: '',
+      },
+      modalDetail: {
+        type: String,
+        required: false,
+        default: '',
+      },
+    },
+  };
+
+</script>
```

Comment:
This component feels like a lot of extra complexity just to set default `continue` and `cancel` labels on the buttons. Maybe consider just using a standard `KModal`?